.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,113 @@
+        - OpenTelemetry.Instrumentation.AWSLambda
+        - OpenTelemetry.Instrumentation.Cassandra
+        - OpenTelemetry.Instrumentation.ElasticsearchClient
+        - OpenTelemetry.Instrumentation.EntityFrameworkCore
+        - OpenTelemetry.Instrumentation.EventCounters
+        - OpenTelemetry.Instrumentation.GrpcCore
+        - OpenTelemetry.Instrumentation.GrpcNetClient
+        - OpenTelemetry.Instrumentation.Hangfire
+        - OpenTelemetry.Instrumentation.Http
+        - OpenTelemetry.Instrumentation.MassTransit
+        - OpenTelemetry.Instrumentation.MySqlData
+        - OpenTelemetry.Instrumentation.Owin
+        - OpenTelemetry.Instrumentation.Process
+        - OpenTelemetry.Instrumentation.Quartz
+        - OpenTelemetry.Instrumentation.Runtime
+        - OpenTelemetry.Instrumentation.SqlClient
+        - OpenTelemetry.Instrumentation.StackExchangeRedis
+        - OpenTelemetry.Instrumentation.Wcf
+        - OpenTelemetry.PersistentStorage.Abstractions
+        - OpenTelemetry.PersistentStorage.FileSystem
+        - OpenTelemetry.ResourceDetectors.AWS
+        - OpenTelemetry.ResourceDetectors.Azure
+        - OpenTelemetry.ResourceDetectors.Container
+        - OpenTelemetry.ResourceDetectors.Host
+        - OpenTelemetry.ResourceDetectors.Process
+        - OpenTelemetry.ResourceDetectors.ProcessRuntime
+        - OpenTelemetry.Sampler.AWS
+        - OpenTelemetry.SemanticConventions
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Package Version
+      description: List of [all OpenTelemetry NuGet packages](https://www.nuget.org/profiles/OpenTelemetry) and version that you are using (e.g. `OpenTelemetry 1.0.2`)
+      value: |
+        | Package Name                          | Version |
+        | ------------------------------------- | ------- |
+        | OpenTelemetry.Api                     | 1.8.0   |
+        | OpenTelemetry                         | 1.8.0   |
+        | TBD                                   | TBD     |
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Runtime Version
+      description: What .NET runtime version did you use? (e.g. `net462`, `net48`, `netcoreapp3.1`, `net6.0` etc. You can find this information from the `*.csproj` file)
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: What happened? Please provide as much detail as you reasonably can.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Create a self-contained project using the template of your choice, apply the minimum required code to result in the issue you are observing. We will close the issue if the repro project you share with us is complex or we cannot reproduce the behavior you are reporting. We cannot investigate custom projects, so don't point us to such, please.
```

Comment:
I think that, link to this page could be useful https://stackoverflow.com/help/minimal-reproducible-example

Reply:
Let me add that, thanks for the suggestion!